packages/design-system/src/components/IdleTimeout/IdleTimeout.tsx

```diff
@@ -77,70 +78,148 @@ const defaultMessageFormatter = (timeTilTimeout: number): React.ReactNode => {
   );
 };
 
+// names for local storage variables
+const timeoutCookieName = 'CMS_DS_TIMEOUT';
+const timeoutWarningCookieName = 'CMS_DS_WARNING';
+
 const IdleTimeout = ({
-  closeDialogText = 'Close',
   continueSessionText = 'Continue session',
   heading = 'Are you still there?',
   endSessionButtonText = 'Logout',
+  endSessionRedirectUrl = '/logout',
```

Comment:
Did you say you were going to remove the default value for this?

Reply:
I'm not sure. I removed redirecting to logout as default behavior for the `onTimeout`, but for the end session url, I kind of assumed that the 'end session' option would only be used in logged in experiences?

That is an assumption though, because I couldn't think through what ending a session on an unauthenticated page would lead to

Reply:
I was thinking when I said this that the presence of this value (OR the on-click handler) controlled whether that button got shown, but I realize now that there's a separate flag for that. The front-end-only (unauthenticated) apps that I know of that currently have one of these timeouts clears local browser storage on timeout, FYI.